packages/block-library/src/navigation/block-navigation-list.js

```diff
@@ -30,12 +129,15 @@ export default function BlockNavigationList( { clientId } ) {
 	} = useDispatch( 'core/block-editor' );
 
 	return (
-		<__experimentalBlockNavigationList
-			blocks={ [ block ] }
-			selectedBlockClientId={ selectedBlockClientId }
-			selectBlock={ selectBlock }
-			showNestedBlocks
-			showAppender
-		/>
+		<NavigableTreeGrid>
+			<__experimentalBlockNavigationList
```

Comment:
Should `<NavigableTreeGrid>` be inside `<__experimentalBlockNavigationList>`? When would you want to use `<__experimentalBlockNavigationList>` and **not** have the grid functionality?